src/SFA.DAS.ApprenticeAccounts.Jobs/EventHandlers/Simulator/SimulateEvents.cs

```diff
@@ -38,5 +41,39 @@ internal class SimulateEvents
                 return new BadRequestObjectResult(e);
             }
         }
+
+        [FunctionName("HandleApprenticeshipCreatedEventTrigger")]
+        public Task<IActionResult> ApprenticeshipCreatedEvent(
+            [HttpTrigger(AuthorizationLevel.Function, "POST", Route = "test-apprenticeship-created-event")] HttpRequestMessage req,
+            ExecutionContext executionContext,
+            ILogger log)
+            => Simulate<ApprenticeshipCreatedEvent>(req, executionContext, log);
+
+        [FunctionName("HandleApprenticeshipUpdatedEventTrigger")]
+        public Task<IActionResult> ApprenticeshipUpdatedEvent(
+            [HttpTrigger] HttpRequestMessage req, ExecutionContext executionContext, ILogger log)
+            => Simulate<ApprenticeshipUpdatedApprovedEvent>(req, executionContext, log);
+
+        [FunctionName("TestApprenticeshipStopped")]
+        public Task<IActionResult> ApprenticeshipStoppedEvent(
+                    [HttpTrigger] HttpRequestMessage req, ExecutionContext executionContext, ILogger log)
+                    => Simulate<ApprenticeshipStoppedEvent>(req, executionContext, log);
+
+        public async Task<IActionResult> Simulate<T>(HttpRequestMessage req, ExecutionContext executionContext, ILogger log)
+        {
+            try
+            {
+                var @event = JsonConvert.DeserializeObject<T>(await req.Content.ReadAsStringAsync());
+
+                await endpoint.Publish(@event, executionContext, log);
+
+                return new AcceptedResult();
+            }
+            catch (Exception e)
+            {
+                return new BadRequestObjectResult(e);
+            }
+        }
+
```

Comment:
These look like events taken from CMAD Jobs, most of which we don't consume. However, it's a security concern creating this Simulate endpoint (as it is a http endpoint, I think).  We spoke to someone about it sometime in the past and you you find a message publisher project for sending the message.